spec/models/framework/definition_spec.rb

```diff
@@ -10,21 +10,13 @@
         it 'returns that framework' do
           expect(definition.framework_short_name).to eql(framework_short_name)
         end
-
-        it 'reports the management charge' do
```

Comment:
This is a nitpick about the commit message: "We no longer need the calculator to report the percentage" - I think this was meant to say "the definition"?

Reply:
I think the message is correct - these specs were verifying that the calculators returned their percentage rate, which we were previously using in the data export for finance. That became unnecessary when we stopped reporting the percentage in the finance report [here](https://github.com/dxw/DataSubmissionServiceAPI/commit/f446b201d1048e95a42b189cd4cf5feb3d6a6d14), which makes these tests not particularly useful or valuable.